Makefile

```diff
@@ -221,6 +221,11 @@ testground:
 .PHONY: testground
 BINS+=testground
 
+wallet-monitor:
+	go build -o wallet-monitor ./cmd/wallet-monitor
+.PHONY: wallet-monitor
+BINS+=wallet-monitor
```

Comment:
To keep with the pattern I would say this should be `lotus-wallet-monitor`.

Reply:
I wonder if it would maybe be better to call this `lotus-monitor`, which would let us expand this as a general utility.

I'd really love to be able to just run `lotus-monitor config.toml` on my miner, with the config containing node API infos, and what to monitor on each node - wallet balances, miner power, deal counts, sector states, etc, etc.